src/main/java/view/InputView.java

```diff
@@ -0,0 +1,37 @@
+package view;
+
+import constant.Exception;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class InputView {
+
+    private static final String DELIMITER = ",";
+    private static final Pattern FINISH_WITH_DELIMITER_REGEX = Pattern.compile(".*,$");
+    private final Scanner scanner = new Scanner(System.in);
```

Comment:
상수와 필드를 분리해 소소한 가독성을 챙겨봅시다 !
```suggestion
    private static final Pattern FINISH_WITH_DELIMITER_REGEX = Pattern.compile(".*,$");
    
    private final Scanner scanner = new Scanner(System.in);
```